portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,16 +54,8 @@
     int numberOfCommentsToDisplay = 0;
     
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
-      //Get number of comments.
-      numberOfCommentsToDisplay = getNumberOfCommentsToDisplay(request);
```

Comment:
Did you delete this for now and then you'll add another way to receive the user's # of comment preference next?